### PR TITLE
v6.0 Release

### PR DESCRIPTION
**Pull request by jennifer-shehane**
For https://github.com/cypress-io/cypress/pull/8437 - 6.0 features in develop also

*Targeted release date: Nov 23, 2020*

- [ ] Migration Guide
- [x] Changelog
- [x] doc updates for opacity: 0 visibility check https://github.com/cypress-io/cypress-documentation/pull/3071
- [x] Rename cy.route2 to cy.http https://github.com/cypress-io/cypress-documentation/pull/3322
- [x] Document printing Electron & Node version https://github.com/cypress-io/cypress-documentation/pull/3324
- [x] Document printing individual component versions https://github.com/cypress-io/cypress-documentation/pull/3339
- [x] Deprecate `experimentalFetchPolyfill` https://github.com/cypress-io/cypress-documentation/pull/3330
- [x] cy.wait() type change https://github.com/cypress-io/cypress-documentation/pull/3337
- [x] rename cy.http() to cy.intercept() https://github.com/cypress-io/cypress-documentation/pull/3340
- [x] Changes to docs necessary from JSON parsing change https://github.com/cypress-io/cypress/pull/9280
- [ ] Add on links https://github.com/cypress-io/cypress-services/pull/3246

### CODE REVIEW

File: source/_partial/network_stubbing_warning.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
...
+{% note warning %}
+⚠️ `cy.route()` and `cy.server()` only support intercepting XMLHttpRequests. Requests using the Fetch API and other types of network requests like page loads and `<script>` tags will not be intercepted by `cy.route()` and `cy.server()`.
 
-Cypress also has a new experimental [route2](/api/commands/route2.html) feature that supports requests using the Fetch API and other types of network requests like page loads. For more information, check out the [cy.route2() documentation](/api/commands/route2.html).{% endnote %}
+**To support requests using the Fetch API you can use one of the solutions below:**
 
+- Use [`cy.http()`](/api/commands/http.html) which supports requests using the Fetch API and other types of network requests like page loads. See [`cy.http()`](/api/commands/http.html).
+- Polyfill `window.fetch` to spy on and stub requests using `cy.route()` and `cy.server()` by enabling [`experimentalFetchPolyfill`](https://on.cypress.io/experimental). See {% issue 95 %} for more details and temporary workarounds.
+{% endnote %}
```
Comment (brian-mann) [inline source/_partial/network_stubbing_warning.md:8]: With `cy.http()` becoming official, I believe these deprecation methods need to be more prominent. It seems odd to list multiple "workarounds" when there is an officially supported version. I suggest removing the `experimentalFetchPolyfill` and explain that `cy.http()` can now achieve this directly.

File: source/_partial/xhr_stubbing_deprecated.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{% note warning %}
+⚠️ **`cy.server()` and `cy.route()` are deprecated in Cypress 6.0.0**. In a future release, support for `cy.server()` and `cy.route()` will be moved to a plugin. Consider using [`cy.http()`](/api/commands/http.html) instead.
```
Comment (brian-mann) [inline source/_partial/xhr_stubbing_deprecated.md:2]: see comment above. not worth calling out `cy.route()` as a plugin... just straight up suggest using `cy.http()`

File: source/_partial/network_stubbing_warning.md (hunk 1 above)
Comment (jennifer-shehane) [reply source/_partial/network_stubbing_warning.md:8]: yah, I agree

File: source/_partial/xhr_stubbing_deprecated.md (hunk 2 above)
Comment (jennifer-shehane) [reply source/_partial/xhr_stubbing_deprecated.md:2]: I'm not sure I agree - unless we are very unsure about future support in a plugin.. The idea of us removing support forever may be intimidating to some users

**Pull request merged**